src/Features/Core/Portable/DocumentationComments/AbstractDocumentationCommentFormattingService.cs

```diff
@@ -59,6 +79,65 @@ public void AppendParts(IEnumerable<TaggedText> parts)
                 _anyNonWhitespaceSinceLastPara = true;
             }
 
+            public void PushList(DocumentationCommentListType listType)
+            {
+                _listStack.Add((listType, 0, false));
+                MarkBeginOrEndPara();
+            }
+
+            public void NextListItem()
```

Comment:
doc.  is this supposed to be called before starting the next list item?  after finishing the next list item?  something else entirely?